From b0303ad5e3b61ca7383892801116abbc5576ec10 Mon Sep 17 00:00:00 2001
From: Numan Siddique <nusiddiq@redhat.com>
Date: Mon, 3 Jun 2019 16:31:08 +0530
Subject: [PATCH] downstream only: Change the version from 2.11.90 to 2.11.1

For fedora only.

Signed-off-by: Numan Siddique <nusiddiq@redhat.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index a9f0a06dc..5862d6a25 100644
--- a/configure.ac
+++ b/configure.ac
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 AC_PREREQ(2.63)
-AC_INIT(openvswitch, 2.11.90, bugs@openvswitch.org)
+AC_INIT(openvswitch, 2.11.1, bugs@openvswitch.org)
 AC_CONFIG_SRCDIR([datapath/datapath.c])
 AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_AUX_DIR([build-aux])
-- 
2.21.0

